docs/integration/alerts-webhooks-custom-params.md

```diff
@@ -0,0 +1,21 @@
+title: Custom Webhooks Parameters
+description: Sematext threshold, anomaly and / or heartbeat Alerts integration with generic webhooks.
+
+Sematext Notification Hooks support a set of variables you can use to customize the information sent from Sematext to the destination of your choice. 
+
+The following variables are supported:
+
+ * **$webHookName** - the name of the webhook
+ * **$applicationId** - application identifier
+ * **$applicationToken** - application token (be careful about providing one as it may allow unauthorized users to send or retrieve data for an App)
+ * **$ruleType** - the type of the Alert Rule
```

Comment:
Btw. what are the types of rules?

Reply:
Mentioned the possible values. I'm not sure if we should describe each and every. What do you think @otisg ?